documentation/slate/source/includes/welcome.md

```diff
@@ -13,10 +13,7 @@ It provide the following services
 * exploration of public transport data / search places
 * and sexy things such as isochrones
 
-navitia is a RESTful API that returns JSON formated results.
-The API has been built on the [HATEOAS model](http://en.wikipedia.org/wiki/HATEOAS) so it should be quite self explanatory since the interactions are defined in hypermedia.
-
-Our APIs are available at the following url: <https://api.navitia.io/v1> (the latest API version is ``v1``).
+navitia is a RESTful API that returns JSON formated results, available at <https://api.navitia.io>
```

Comment:
Is there no reference to HATEOAS anymore? I think it's nice to advertise about it.